kids/lib/screens/settings/applanguage.dart

```diff
@@ -0,0 +1,101 @@
+import 'package:auto_route/auto_route.dart';
+import 'package:brunstadtv_app/theme/design_system/design_system.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_hooks/flutter_hooks.dart';
+import 'package:flutter_svg/svg.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+import 'package:kids/helpers/svg_icons.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:responsive_framework/responsive_breakpoints.dart';
+import 'package:kids/components/settings/applanguage_list.dart';
+import 'package:brunstadtv_app/helpers/languages.dart';
+
+@RoutePage<void>()
+class AppLanguageScreen extends HookConsumerWidget {
+  const AppLanguageScreen({super.key});
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    
+    final design = DesignSystem.of(context);
+    final bp = ResponsiveBreakpoints.of(context);
+    final double basePadding = bp.smallerThan(TABLET) ? 24.0 : 48.0;
+    final selectedLanguageIndex = useState(0); 
+
+    List<ApplanguageListItem> buildItem() {
+      List<ApplanguageListItem> itemList = [];
+      for (int i = 0; i < appLanuageCodes.length; i++) {
```

Comment:
Instead of a for loop, you can simplify this with `appLanuageCodes.map((code) => ApplanguageListItem(...))`